tests/test_initialization.py

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import unittest
-
+import pytest
```

Comment:
Haven't checked thoroughly but have we started migrating to `pytest` in `peft`?

Reply:
From my understanding, we don't really plan on migrating. Instead, we can consider using more pytest features if we add new tests or are updating existing tests. Here I thought it would only minimally change the code, so doing this change should be fine.